packages/core/scripts/admin-proposals/simulateVote.js

```diff
@@ -0,0 +1,247 @@
+// Description:
+// - Simulate voting affirmatively on any pending Admin Proposals
+
+// Run:
+// - For testing, start mainnet fork in one window with `yarn hardhat node --fork <ARCHIVAL_NODE_URL> --no-deploy`
+// - This script should be run after any Admin proposal UMIP script against a local Mainnet fork. It allows the tester
+// to simulate what would happen if the proposal were to pass and to verify that contract state changes as expected.
+// - Vote Simulate: HARDHAT_NETWORK=localhost node ./packages/core/scripts/admin-proposals/simulateVote.js
+
+const hre = require("hardhat");
+require("dotenv").config();
+const { GasEstimator } = require("@uma/financial-templates-lib");
+const winston = require("winston");
+const {
+  interfaceName,
+  advanceBlockAndSetTime,
+  isAdminRequest,
+  getRandomSignedInt,
+  computeVoteHash,
+  signMessage,
+} = require("@uma/common");
+const { _getContractAddressByName, _impersonateAccounts } = require("./utils");
+
+// By default, connect to localhost provider:
+const DEFAULT_PROVIDER = "http://127.0.0.1:8545";
```

Comment:
Shouldn't we just grab our web3 instance from hardhat rather than initializing it manually using `hre.web3`? I think that's what `getContract` will be relying on.

Reply:
Ah yes this looks like a leftover variable I'll remove.